### DISCUSSION TITLE
DialogOptionsEx Upgrade Issue

### DISCUSSION THREAD

**Discussion by Sharafudeen** [fgilde/MudBlazor.Extensions#95] [General]
Hi,

I'm getting below error after upgraded MudBlazor 6.20.0 to 7.0.0 and MudBlazor.Extensions 1.7.89 to 2.0.0-prev-2407080832-main 

[DialogOptionsEx Error](https://github.com/fgilde/MudBlazor.Extensions/assets/42769344/6a96884e-fc26-48ee-b34e-549d1a2a9219)

**Comment by Sharafudeen** [comment]
Removed all artifacts (bin & obj folders) and rebuild the project solution.

**Reply by fgilde** [reply]
and now it works? or error still there? I didn't have this error. Try incognito tab, maybe its a cache problem from a js

**Reply by Sharafudeen** [reply]
> and now it works? or error still there? I didn't have this error. Try incognito tab, maybe its a cache problem from a js

Yes it is working but Glasses Theme like Dialog is not working. 

**Note:** Mudex.org having this theme for About dialog.
